src/Gateways/Handlers/PaymentActionHandler.php

```diff
@@ -24,7 +24,7 @@ public function handle() {
 
 		$this->save_meta_to_order( $this->request->order, array( 'payment_action' => $txn_type ) );
 
-		if ( AbstractGateway::TXN_TYPE_VERIFY !== $txn_type ) {
+		if ( AbstractGateway::TXN_TYPE_SALE === $txn_type ) {
```

Comment:
Why are we no longer treating sales and authorizations the same? Historically, we've considered auths complete during checkout.

Reply:
I've reverted the change. Although, it's a little bit odd that there is not even a slightly way to differentiate between an order that was charged and one that needs further action. The capture option appears in both setups.